src/pages/index.js

```diff
@@ -17,6 +17,9 @@ import { RequisitionsPage } from './RequisitionsPage';
 import { RequisitionPage,
          checkForFinaliseError as checkForRequisitionFinaliseError,
        } from './RequisitionPage';
+import { SupplyRequisitionPage,
+         checkForFinaliseError as checkForSupplierRequisitionError,
```

Comment:
Yeah the FinaliseModal will run a checkForError function if it exists, otherwise it will just allow finalising without any check